ocrd_webapi/main.py

```diff
@@ -50,6 +56,13 @@ async def startup_event():
     """
     await initiate_database(DB_URL)
 
+    # If the default user account is not available
+    # in the DB, create it
+    try:
+        await authenticate_user('test', 'test')
+    except ValueError:
+        await register_user('test', 'test')
```

Comment:
Kba said it already I think the env's should be used here so that the service-creator can use the credentials (OCRD_WEBAPI_USERNAME, _PASSWORD) right away. In addition `validated_user` should be set for this "default-user"

Reply:
I also thought about that but many users may be created on the fly when the environment variables change unless the default user account is terminated when the server shuts down.

Reply:
Previously you could set OCRD_WEBAPI_USERNAME and OCRD_WEBAPI_PASSWORD and then use that for authentication. But now that is completely gone. Now you have to use 'test':'test' and that is hard coded and easy to guess.

Reply:
> Previously you could set OCRD_WEBAPI_USERNAME and OCRD_WEBAPI_PASSWORD 

It will be replaced with that. For convenience, it is `test` hardcoded here. Previously nothing was stored in the DB regarding the users. 

The question here is more about do we keep the default accounts created by the environment variables or remove them once the server shuts down. If we don't remove them and the same DB is used and different env variables for authentication are set, then different default admin accounts will be created on the fly and they should be manually deleted from the DB.